tests/js/dataMock/analytics.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * Analytics test data mock.
```

Comment:
@felixarntz and I discussed putting this in the module's utilities similar to the current fixtures and factories which I think makes more sense than in a higher-level test path since this is where we will be reaching for it the most.

I'm open to alternate directories rather than `utils` though (perhaps a new `__data__`?) but I think we should keep it within `assets/js/modules/analytics/`. It won't be included in builds though since it will never be imported outside of Storybook or tests. Keep in mind that Storybook stories will start to be be co-located with the components themselves so it makes sense that this kind of thing would also be within the module.

Reply:
Updated.